toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -122,6 +52,27 @@ var (
 	signatureHandling    = app.Flag("signature-handling", "Specifies how to handle signature mismatches for source files.").Default(signatureEnforceString).PlaceHolder(exe.PlaceHolderize(validSignatureLevels)).Enum(validSignatureLevels...)
 )
 
+func populateSrpmPackerConfig() *srpmpacker.Config {
```

Comment:
Similar request here and all other commits - let's keep `main` as the first function.